c10/util/Exception.cpp

```diff
@@ -114,6 +114,13 @@ void torchInternalAssertFail(
   torchCheckFail(func, file, line, c10::str(condMsg, userMsg));
 }
 
+bool env_flag_set(const char* env_var_name) {
+  const char* const env_string = std::getenv(env_var_name);
+  return (env_string == nullptr) ? false : std::strcmp(env_string, "0");
+}
+
+C10_EXPORT bool TORCH_RUNTIME_DEBUG = env_flag_set("TORCH_RUNTIME_DEBUG");
```

Comment:
So I haven't seen this pattern anywhere and I feel like there is probably a reason for this. 

I verified with lldb that this is hit once during loading of libc10
``` Shell
frame #0: 0x00007ffff70e0d64 libc10.so`::__cxx_global_var_init.9() at Exception.cpp:122:39
```
std::getenv is labeled no_execpt() but still idk feels a little sketch. I think maybe I export the symbol but just set it to false, and do an initial os.getenv() in torch/__init__.cpp ?